pkg/controller/controller.go

```diff
@@ -168,5 +190,14 @@ func logEvent(obj interface{}, kind, eventType string, err error) {
 
 	// Send notification to communication chennel
 	notifier := notify.NewSlack()
-	notifier.Send(event)
+	notifier.SendEvent(event)
+}
+
+func sendMessage(msg string) {
```

Comment:
Keep this function in notifier

Reply:
@ssudake21 we already have `notifier.SendMessage(msg)` function in notifier package